### PR TITLE
Add velero-volume-controller as an implemented choice of velero restic 3rd party controller

### LINKED ISSUE #2375: Add 3rd party controller to build an automatic way to backup all volume resources in the cluster
**Describe the problem/challenge you have**
Refers to #2355 
> > Is there a do-one-command or automatic way for us to backup all persistent volume resources in the cluster? Right now, we have to add relevant annotations to all of pods with pv resources in cluster, which is complicated and unpractical in a production environment.
> 
> Velero doesn't have a built-in command to do this, you'd need to write your own script/automation. You could check out this community project: https://github.com/bitsbeats/velero-pvc-watcher

**Describe the solution you'd like**
Build an automatic way to backup all volume resources in the cluster

**Anything else you would like to add:**
@skriss To solve this, I have created a repo called [velero-volume-controller](https://github.com/duyanghao/velero-volume-controller), which is a Kubernetes controller for velero that detects and adds relevant backup annotation to pods with volumes. Could you take a look at this and add it to the 3rd party controller part in the [velero Restic Integration docs](https://velero.io/docs/v1.3.1/restic/)?

velero-volume-controller

### PR DESCRIPTION
**Pull request by duyanghao**
Closes #2375 

[velero-volume-controller](https://github.com/duyanghao/velero-volume-controller) is a Kubernetes controller for velero that detects and adds relevant backup annotation to pods with volumes, which helps to build an automatic way to backup all volume resources in the cluster.

Signed-off-by: duyanghao <1294057873@qq.com>

### CODE REVIEW

File: site/docs/master/restic.md [hunk 1]
```diff
@@ -418,6 +418,12 @@ Velero does not currently provide a mechanism to detect persistent volume claims
 
 To solve this, a controller was written by Thomann Bits&Beats: [velero-pvc-watcher][7]
 
+### Add backup annotation
+
+Velero hasn't provided a do-one-command or automatic way to backup all volume resources in the cluster without annotation.
+
+To solve this, [velero-volume-controller][10] helps to achieve this goal by adding backup annotation to pods with volumes automatically.
```
Comment (carlisia) [inline site/docs/master/restic.md:425]: Looks good. It'd be helpful if you added something like "written by @duyanghao (or a more accurate designation) somewhere in this line.

File: site/docs/master/restic.md [hunk 2]
```diff
@@ -418,6 +418,12 @@ Velero does not currently provide a mechanism to detect persistent volume claims
 
 To solve this, a controller was written by Thomann Bits&Beats: [velero-pvc-watcher][7]
 
+### Add backup annotation
+
+Velero hasn't provided a do-one-command or automatic way to backup all volume resources in the cluster without annotation.
```
Comment (nrb) [inline site/docs/master/restic.md:423]: I would reword this slightly, changes in bold.

> Velero **does not currently provide a single command** or automatic way to backup all volume resources in the cluster without **annotating pods or pod templates**.

File: site/docs/master/restic.md (hunk 1 above)
Comment (nrb) [inline site/docs/master/restic.md:425]: Some changes I would suggest in bold:

>The [velero-volume-controller][10] **helps to solve this problem** by adding **the** backup annotation to pods with volumes automatically.

I also agree with Carlisia's request for attribution.

**Comment by duyanghao** [comment]
@carlisia @skriss @nrb Thanks for your comments, I have changed it as you suggest.

**Comment by skriss** [comment]
@duyanghao since we've released v1.3.2 in the interim, you probably want to make this change in the `site/docs/v1.3.2` as well, but it's not 100% essential since you got it into `site/docs/master`. I'll approve this PR, but feel free to make that addition.
